### ISSUE TITLE
Trained model

### ISSUE THREAD

**Issue by Horkyze** [danielegrattarola/twitter-sentiment-cnn#15]
Hey again, I have the fully trained model at my disposal (it took whole night to train). It has 406 MB (uncompressed).
I've uploaded it to mega as zip. If you want I can create a PR to master or other branch so other people have easy access to the complete solution without the need of training the network. 

Anyway here is the link https://mega.nz/#!xVg0ARYK!oVyBZatotQGOD_FFSzZl5gTS1Z49048vjFEbyzftcFY

**Comment by danielegrattarola** [comment]
Hi! Thank you so much, we should definitely include it in the repo. I don't want to push a 406MB object to GitHub, so maybe we can just include the link to Mega in the README.
Can you give me more info on the trained model's performance? Things like training and validation loss and accuracy.
Cheers

**Comment by Horkyze** [comment]
Honestly, this is the first time I tried to play with machine learning/neural nets, so I am not really sure how to evaluate the performance. But if you guide me, I surely can ;)

**Comment by danielegrattarola** [comment]
I checked the data you shared and all the info I needed was in the logfile.  
What I was looking for were the validation accuracy and loss of the model, which tell us how well is the model performing on unseen data. I've added the link to mega in the README as well as a link to your username. Let me know if you would like anything changed or more credit given.

Thanks for sharing.
Cheers

**Comment by Horkyze** [comment]
Awesome! Its all good, thanks ;)
